Champions/Karthus/Q.cs

```diff
@@ -0,0 +1,72 @@
+using GameServerCore.Enums;
+using GameServerCore.Domain.GameObjects;
+using static LeagueSandbox.GameServer.API.ApiFunctionManager;
+using GameServerCore.Domain;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+
+namespace Spells
+{
+    public class KarthusLayWasteA1 : IGameScript
+    {
+        public void OnActivate(IChampion owner)
+        {
+        }
+
+        public void OnDeactivate(IChampion owner)
+        {
+        }
+
+        public void OnStartCasting(IChampion owner, ISpell spell, IAttackableUnit target)
+        {
+            AddParticleTarget(owner, "Karthus_Base_Q_Hand_Glow.troy", owner, 1, "R_Hand");
+            AddParticle(owner, "Karthus_Base_Q_Point.troy", spell.X, spell.Y);
+            AddParticle(owner, "Karthus_Base_Q_Ring.troy", spell.X, spell.Y);
+            AddParticle(owner, "Karthus_Base_Q_Skull_Child.troy", spell.X, spell.Y);
+            CreateTimer(0.5f, () =>

```

Comment:
I'm not sure if that should be a timer at all. There are functions for `FinishCasting` and `ApplyEffects`. Please check if some of these functions will help you removing this timer.

Reply:
All fixed.